### PR TITLE
Gulp Structure Updates

### PR DESCRIPTION
**Pull request by RobertGrubb**
- Defined all requires in gulpfile.js as global
- Defined paths as global, and prefixed with underscore to separate from
tasks
- Removed all requires in individual task files.

### CODE REVIEW

File: tasks/clean.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
-var gulp = require('gulp');
-var del = require('del');
-var paths = require('./paths');
-
 gulp.task('clean', function() {
+  console.log(paths.output.build);
```
Comment (bretthayes) [inline tasks/clean.js]: Remove console.log

File: gulpfile.js [hunk 2]
```diff
@@ -1,2 +1,25 @@
+/**
+ * Setting all of the variables here without 'var'
```
Comment (bretthayes) [inline gulpfile.js]: Although this is super awesome and removes duplication of using requires in all of our tasks, I'm not sure about using globals. Hold off on merging just yet and let's do some research.

Comment (RobertGrubb) [inline gulpfile.js]: Definitely agree, I'm not quite sure if it's considered "bad practice" or not.

Comment (bretthayes) [inline gulpfile.js]: It sort of ruins the point of require's, but then again, it's a build system and not application logic.

**Comment by bretthayes** [comment]
LGTMpc

**Pull request merged**